web/views.py

```
@@ -6469,6 +6473,127 @@
+        submissions = paginator.page(paginator.num_pages)
+
+    context = {
+        "submissions": submissions,
+        "viewing_as": viewing_as,
+        "pdf_types": PDFType.objects.all(),
+    }
+    return render(request, "web/pdf/submission_list.html", context)
+
+
+@login_required
+def pdf_submission_detail(request, submission_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `pdf_submission_detail`

(ANN201)